### DISCUSSION TITLE
Archery配置实例测试连接时，Redis和Mongo遇到的问题

### DISCUSSION THREAD

**Discussion by ChengJiuSu** [hhyo/Archery#1667] [Q&A] [answered]
由于使用的是AWS 的 Redis 和 Mongo 的atles测试连接时遇到了一些问题，希望有大佬可以支持处理一下
问题如下：
测试连接redis
返回 无法连接实例,”\nunknown command CONFIG, with args beginning with: GET, databases“

MongoDB，使用的是云数据库，并且是多副本的方式，在配置实例连接的时候，实在不清楚应该怎么配置，各种方式都进行过尝试，但效果不大
例如：
实例连接配置：
test-atlas-pre-version-cluster-pl-0.j3vas.mongodb.net
或者
pl-0-us-west-2.j3vas.mongodb.net.:1025,pl-0-us-west-2.j3vas.mongodb.net.:1024,pl-0-us-west-2.j3vas.mongodb.net.:1026、
pl-0-us-west-2.j3vas.mongodb.net.:1025,pl-0-us-west-2.j3vas.mongodb.net.:1024,pl-0-us-west-2.j3vas.mongodb.net.:1026/myFirstDatabase?ssl=true&replicaSet=atlas-slgwft-shard-0&authSource=admin
再或者
pl-0-us-west-2.j3vas.mongodb.net 
等配置方式都无效

以上问题 希望有大佬能过技术支持，万分感谢

**Comment by hhyo** [comment, marked as answer]
redis的这个检测是已知问题，后续需要看有没有其他检测方式可以加上

mongodb可以提供下错误日志，看下具体的原因

**Reply by ChengJiuSu** [reply]
无法连接实例，
connection closed, Timeout: 30s, Topology Description:
<TopologyDescription id: 62cb80b5b4e1ccc7061cfc8d,
topology_type: Single, servers: [«ServerDescription ('pI-O-us-
west-2.j3v1w.mongodb.net', 1026) server_type: Unknown, rtt:
None, error=AutoReconnect ('connection closed')>]
这是连接实例时web ui 输出的报错；观察了日志文件没有时间日志输出。
并且连接方式是以：
![image](https://user-images.githubusercontent.com/108964694/178173847-ec77f742-4627-467a-8176-2955a83d2122.png)
方式进行连接的

**Reply by ChengJiuSu** [reply]
通过代码复现也是相同报错；所以想确认一下 目前是否不支持mongo副本级集群使用

**Comment by ChengJiuSu** [comment]
是的，redis 是因为 aws 官方禁用了config命令，我在自建redis数据库也进行了复现，确实是需要调用config命令；
mongo的话，除了web ui 有错误日志输出，但没有错误日志输出到容器日志里
